layouts/shortcodes/knockout-scores.html

```diff
@@ -0,0 +1,31 @@
+{{ $event := index .Site.Data.events (.Get 0) }}
+{{ $match_points := $event.match_points }}
+
+{{ range $match := $match_points }}
+    {{ if eq .type "knockout" }}
```

Comment:
Rather than wrapping the contents in an `if`, it would be nicer to use https://gohugo.io/functions/where/#readout on the `range`, something like:

`{{ range $match := where $match_points ".type" "knockout" }}`

Reply:
c42cd40